javascript/javascript1/week3/warmup2.js

```diff
@@ -0,0 +1,14 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function travelDuration(info){
+  const time = info.destinationDistance / info.speed;
+  const hour = Math.floor (time);
+  const min = Math.trunc((time - hour)*60);
+  return `${hour} hours and ${min} minutes`
```

Comment:
The variables are generally well-named here, but `time` could do with a unit and `min` can be misunderstood as `minimum`. 👀 

To be a little annoying, if something took exactly 1 minute, you would return `0 hours and 1 minutes`; can you think of a way to improve this? 😄

Reply:
Good point! 😊 I've updated the return statement so that if the hours are 0, it will return the minutes; otherwise, it will include both hours and minutes. I also renamed the variables for clarity, changing hours to totalHours and min to timeInMinutes.